### PR TITLE
tf quickstart yarn integration test

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

**Comment by yangw1234** [comment]
Does it work? 
Here are a few steps might be necessary for tensorflow to acess HDFS.
https://github.com/intel-analytics/analytics-zoo/tree/branch-0.9/pyzoo/zoo/examples/orca/learn/tf/lenet#run-examples-on-yarn-cluster

**Comment by Le-Zheng** [comment]
jenkins pass: http://10.112.231.51:18888/view/ZOO-PR/job/ZOO-PR-Python-integration-test/579/console

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md [hunk 1]
```diff
@@ -38,7 +38,13 @@ elif cluster_mode == "yarn":  # For Hadoop/YARN cluster
 
 This is the only place where you need to specify local or distributed mode. View [Orca Context](./../Overview/orca-context.md) for more details.
 
-**Note:** You should `export HADOOP_CONF_DIR=/path/to/hadoop/conf/dir` when running on Hadoop YARN cluster. View [Hadoop User Guide](./../../UserGuide/hadoop.md) for more details.
+**Note:** You should correctly set HADOOP_HOME, HADOOP_CONF_DIR, HADOOP_HDFS_HOME, LD_LIBRARY_PATH, etc when running on Hadoop YARN cluster.
```
Comment (yangw1234) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md:41]: How about, "To use tensorflow_datasets on HDFS, you should correctly set HADOOP_HOME, HADOOP_CONF_DIR, HADOOP_HDFS_HOME, LD_LIBRARY_PATH, etc. For more details, please refer to TensorFlow documentation [link](https://github.com/tensorflow/docs/blob/r1.11/site/en/deploy/hadoop.md),"

Comment (Le-Zheng) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md]: updated.

**Comment by yangw1234** [comment]
LGTM

**Pull request merged**